plugins/coverbrowser.koplugin/bookinfomanager.lua

```diff
@@ -691,7 +691,6 @@ function BookInfoManager:extractInBackground(files)
             local cover_specs = files[idx].cover_specs
             logger.dbg("  BG extracting:", filepath)
             self:extractBookInfo(filepath, cover_specs)
-            FFIUtil.usleep(100000) -- give main process 100ms of free cpu to do its processing
```

Comment:
You're sure ? This was mostly to let the main process time to sql query, fetch some data and cover, process the cover, force a repaint/refresh, and the whole thing feeling smooth. You think the kernel multitasking is enough, even on single cpu devices ?

Reply:
Seeing the base PR and kernel priority stuff. In my experience on normal linux servers (when I had 1/2 cores), I just don't trust nice/ionice to give enough smoothness (I use them because why not give us as much chances, but I often end up adding a sleep somewhere if I can when i need smoothness).

Reply:
It's definitely more than enough. The main process is not going to be doing anything until the next scheduled task, which is the item update scheduled every second.

That's extremely minimal contention to begin with, even if extractions hasn't finished yet.

Reply:
A NORMAL process at nice 0 can can run circles around a BATCH process at nice 5 ;).

Reply:
Yep, behaves perfectly fine even on a crappy H2O ;).